mailbox/tools/indexer/src/test/java/org/apache/mailbox/tools/indexer/SingleMailboxReindexingTaskSerializationTest.java

```diff
@@ -43,7 +43,7 @@
     private final String serializedMailboxReindexingTask = "{\"type\": \"mailbox-reindexing\", \"mailboxId\": \"1\", \"runningOptions\":{\"messagesPerSecond\":50}}";
     private final String legacySerializedMailboxReindexingTask = "{\"type\": \"mailbox-reindexing\", \"mailboxId\": \"1\"}";
     private final String serializedAdditionalInformation = "{\"type\": \"mailbox-reindexing\", \"mailboxId\": \"1\", \"successfullyReprocessedMailCount\":42,\"failedReprocessedMailCount\":2,\"messageFailures\":[{\"mailboxId\":\"1\",\"uids\":[10, 20]}], \"mailboxFailures\": [\"2\"],\"timestamp\":\"2018-11-13T12:00:55Z\", \"runningOptions\":{\"messagesPerSecond\":50}}";
-    private final String legacySerializedAdditionalInformation = "{\"type\": \"mailbox-reindexing\", \"mailboxId\": \"1\", \"successfullyReprocessedMailCount\":42,\"failedReprocessedMailCount\":2,\"messageFailures\":[{\"mailboxId\":\"1\",\"uids\":[10, 20]}], \"timestamp\":\"2018-11-13T12:00:55Z\"}";
```

Comment:
if you change a name of a field in a legacy test, is it still a legacy test ?

Reply:
(the comment you left is outdated, I suspect it is an history issue.)

Reply:
I think you are viewing the outdated version? Previously I changed the field name from 'failures' to 'messageFailures' then changed it back so no changes after all.

Reply:
> (the comment you left is outdated, I suspect it is an history issue.)

Yes it is an history issue, reviewing the full diff I spotted no changes on legacy stuff.

As written here https://github.com/linagora/james-project/pull/3423#issuecomment-639379956 maybe a rebase could help per-commit reviewers.

Reply:
ok, with github this commit appeared as the last one :/